tests/console/sshd.pm

```diff
@@ -126,16 +129,58 @@ sub run {
     assert_script_run "scp -4v '$ssh_testman\@localhost:/etc/{group,passwd}' /tmp";
     assert_script_run "scp -4v '$ssh_testman\@localhost:/etc/ssh/*.pub' /tmp";
 
+    # poo#80716 Test all available ciphers, key exchange algorithms, host key algorithms and mac algorithms.
+    # TODO: This does not work for Tumbleweed because of nmap
+    # See pull request #11930 for more details
+    test_cryptographic_policies(remote_user => $ssh_testman) unless (is_opensuse);
```

Comment:
While this is a bit unfortunate, I also think it makes sense to run this test on SLE, until we find a solution for openSUSE as well.

Just my two cents, I find this a good PR!